packages/mobile/branding/celo/src/brandingConfig.ts

```diff
@@ -3,5 +3,6 @@ export const CELO_FAUCET_LINK = 'https://celo.org/developers/wallet'
 export const CELO_TERMS_LINK = 'https://celo.org/terms'
 export const TOS_LINK = 'https://celo.org/user-agreement'
 export const FAQ_LINK = 'https://celo.org/faq'
+export const FUNDING_LINK = 'https://valoraapp.com/fund-wallet'
```

Comment:
Question: Is it fine to add a link to valoraapp.com here? I think we wanted to keep links to celo.org here, but there is no link to additional funding options there. I added the link in the branding repo as well: https://github.com/clabs-co/valora-app-branding/pull/4

Reply:
no big deal, but would prefer to see celo.org here even if it doesn't exist.

```suggestion
// Note: the actual link doesn't exist on celo.org and is overridden by the Valora branding
export const FUNDING_LINK = 'https://celo.org/fund-wallet'
```